contracts/parent-chain/contract-upgrades/CelestiaNitroContracts2Point1Point3UpgradeAction.sol

```diff
@@ -129,9 +129,6 @@ contract CelestiaNitroContracts2Point1Point3UpgradeAction {
             proxy: TransparentUpgradeableProxy(payable((sequencerInbox))),
             implementation: isERC20 ? newERC20SequencerInboxImpl : newEthSequencerInboxImpl
         });
-        // Set the new EspressoTEEVerifier address.
```

Comment:
why are we deleting this?

Reply:
This was included in the upgrade action twice if you look right below this removal.